examples/2D_isentropicvortex/case.py

```diff
@@ -0,0 +1,107 @@
+    'run_time_info'                : 'T',
+    # ==========================================================================
+
+    # Computational Domain Parameters ==========================================
+    'x_domain%beg'                 : -5.,
+    'x_domain%end'                 : 5.,
+    'y_domain%beg'                 : -5.,
+    'y_domain%end'                 : 5.,
+    'm'                            : Nx,
+    'n'                            : Nx,
+    'p'                            : 0,
+    'dt'                           : mydt,
+    't_step_start'                 : 0,
+    't_step_stop'                  : int(Nt),
+    't_step_save'                  : int(Nt),
+    # ==========================================================================
+
+    # Simulation Algorithm Parameters ==========================================
+    'num_patches'                  : 1,
+    'model_eqns'                   : 3,
+    'alt_soundspeed'               : 'F',
+    'num_fluids'                   : 1,
+    'adv_alphan'                   : 'T',
+    'mpp_lim'                      : 'F',
+    'mixture_err'                  : 'F',
+    'time_stepper'                 : 3,
+    'weno_order'                   : 3,
+    'weno_eps'                     : 1.E-16,
+    'mapped_weno'                  : 'T',
+    'null_weights'                 : 'F',
+    'mp_weno'                      : 'F',
+    'riemann_solver'               : 2,
+    'wave_speeds'                  : 1,
+    'avg_state'                    : 2,
+    'bc_x%beg'                     : -1,
+    'bc_x%end'                     : -1,
+    'bc_y%beg'                     : -1,
+    'bc_y%end'                     : -1,
+    # ==========================================================================
+
+    # Formatted Database Files Structure Parameters ============================
+    'format'                       : 1,
+    'precision'                    : 2,
+    'prim_vars_wrt'                :'T',
+    'parallel_io'                  :'T',
+    'omega_wrt(3)'                 :'T',
+    'fd_order'                     : 2,
+    # ==========================================================================
+
+    # Patch 1 ==================================================================
+    'patch_icpp(1)%geometry'       : 3,
+    'patch_icpp(1)%x_centroid'     : 0,
+    'patch_icpp(1)%y_centroid'     : 0,
+    'patch_icpp(1)%length_x'       : 10.,
+    'patch_icpp(1)%length_y'       : 10.,
+    'patch_icpp(1)%vel(1)'         : vel1,
+    'patch_icpp(1)%vel(2)'         : vel2,
+    'patch_icpp(1)%pres'           : pres,
+    'patch_icpp(1)%rho'            : 1,
+    'patch_icpp(1)%alpha_rho(1)'   : alpha_rho1,
+    'patch_icpp(1)%alpha(1)'       : 1.,
+    # ==========================================================================
+
+    # Fluids Physical Parameters ===============================================
+    'fluid_pp(1)%gamma'            : 1.4,
```

Comment:
this is not the gamma you want, it is actually capital Gamma. see Coralic and Colonius 14 (and the code documentation).

Reply:
I reverted this back